.github/workflows/cloud-workflow.yml

```
@@ -0,0 +1,175 @@
+---
+name: cloud-workflow
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+    paths:
+      - 'orc8r/**'
+      - 'lte/**'
+      - 'feg/**'
+      - 'cwf/**'
+  pull_request:
+    branches:
+      - master
+jobs:
+  insync-checkin:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
+      - name: Run build.py
+        run: |
+            cd ${MAGMA_ROOT}/orc8r/cloud/docker
+            python3 build.py --generate
+            sudo chown -R $USER $MAGMA_ROOT/*
+            git add .
+            git status
+            git diff-index --quiet HEAD
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action insync-checkin failed"
+          SLACK_USERNAME: "Cloud workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  deploy-sync-checkin:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - name: Install Docker Compose
+        env:
+          DC_VERSION: '1.29.1'
+        run: |
+            sudo curl -L https://github.com/docker/compose/releases/download/${DC_VERSION}/docker-compose-`uname -s`-`uname -m` > ~/docker-compose
+            sudo chmod +x ~/docker-compose
+            sudo mv ~/docker-compose /usr/local/bin/docker-compose
+      - name: Run build.py
+        run: |
+            cd ${MAGMA_ROOT}/orc8r/cloud/deploy/orc8r_deployer/docker
+            ./run_deployer.bash --deploy-dir /tmp/deploy_dir --build --test check_all
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action deploy-sync-checkin failed"
+          SLACK_USERNAME: "Cloud workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  cloud-lint:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
+      - name: Lint cloud Go code
+        run: |
+            cd ${MAGMA_ROOT}/orc8r/cloud/docker
+            python3 build.py --lint
+      - name: Generate test coverage
+        run: |
+            cd ${MAGMA_ROOT}/orc8r/cloud/docker
+            python3 build.py --coverage
+      - name: Notify failure to slack
```

Here and elsewhere: let's be consistent about capitalization etc
```suggestion
      - name: Notify failure to Slack
```

---

fixed